recipes/wineditline/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package C)
+
+find_package(wineditline REQUIRED CONFIG)
+
+include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
+conan_basic_setup()
```

```suggestion
conan_basic_setup(TARGETS)
```
To be sure you are consuming only wineditline from the config cmake file.